kornia/contrib/extract_patches.py

```diff
@@ -9,11 +9,12 @@
 from kornia.core import Module, Tensor, pad
 
 FullPadType = Tuple[int, int, int, int]
-PadType = Union[int, Tuple[int, int], FullPadType]
+TuplePadType = Union[Tuple[int, int], FullPadType]
+PadType = Union[int, TuplePadType]
 
 
 def create_padding_tuple(padding: PadType, unpadding: bool = False) -> FullPadType:
-    padding = _pair(padding)
+    padding = cast(TuplePadType, _pair(padding))
```

Comment:
Should we expose the https://github.com/kornia/kornia/blob/619d65f9caae73016cad33d7aa006be4095a1164/kornia/filters/kernels.py#L26 on kornia core or something? this way we don't need to use this internal ops from torch. @edgarriba